src/Parser/Shaman/Restoration/Modules/Talents/UnleashLife.js

```diff
@@ -251,7 +251,7 @@ class UnleashLife extends Analyzer {
 
   unleashLifeCastRatioChart() {
     const totalUses = this.buffedChainHeals + this.buffedHealingWaves + this.buffedHealingSurges + this.buffedRiptides;
-    const unusedUL = totalUses - this.unleashLifeCasts;
+    const unusedUL = this.unleashLifeCasts - totalUses;
```

Comment:
This looks like a significant change, was this intended?

Reply:
I noticed logs where Unleash Life unused casts showed as negative, on top of never seeing it higher than 0. After investigating, this being the wrong way around was the cause since `11 Buffs used - 12 Casts = -1 Unused`
The change fixes it to `12 Casts - 11 Buffs used = 1 Unused`